src/Illuminate/Support/Str.php

```diff
@@ -1313,7 +1313,7 @@ public static function apa($value)
 
         $words[0] = ucfirst(mb_strtolower($words[0]));
 
-        for ($i = 0; $i < count($words); $i++) {
+        for ($i = 0, $iMax = count($words); $i < $iMax; $i++) {
```

Comment:
Note that `count()` in php is a constant-time function, i.e. it does not literally count the array items each loop. Hence, this probably is not the improvement it could be in other programming languages.

Reply:
PHP does call count on every iteration, you can replace count with custom function and you'll see it being called every time.

Reply:
`count()` is indeed called each time the loop condition is evaluated (once per iteration). My point is, `count()` is a very cheap function (`O(1)` runtime complexity), so there is no need to optimize it away. Introducing another variable to hold this value hence is unnecessary/redundant code. This is an example of micro-optimization one should avoid.